### DISCUSSION TITLE
Moving position for tracking

### DISCUSSION THREAD

**Discussion by Flip2376** [FacilMap/facilmap#249] [Ideas]
Hi, 
at first I have to say that the idea of this software and especially the simplicity is really great. I spend my free time in a volunteer fire brigade and we measure for example natural radioactivity in our city and use facilmap for documentation of fixed measuring points. A great feature could be a special marker "car" which shows the current position of car x or y in the map.  So the result is that a car gets a unique name (e.g. license plate) and is shown for everybody in the map where he is at the moment. Maybe the car sets automatically every x seconds a point with the current position which disappears after few seconds before the next position-marker appears.

Another small thing is that it could be easier in few cases that the name of the marker can be seen the whole time on the map and not only by moving with the mouse over the marker. 

I hope that these ideas can be implemented and again: great software.
many greetings
Flip

**Comment by cdauth** [comment]
Great ideas!

I also have a use case for the live tracking, so I hope to find the time to implement it soon!

**Reply by Flip2376** [reply]
Thank you for your quick reply. We have tested the software quiet often and found few further ideas which could be helpful.
1. It is only possible to export the content of the markers. But we found no possibility to export the settings of the marker. So the idea is to export the marker to another map for testing/creating variations. So it would be great to create a new map and import the settings of the marker and then, if necessary, the content of the marker.
2. The order of the marker is fixed. So whenever I have created a list of different marker and want do add a new type of marker on the second position, I have to delete all later markers and create them new.
3. The default content of the marker which can be seen on the map is "untitled marker". It would be great to have a possibility to set another default value or even better to have the choice to use dropdown list for example...

I hope you find the time to implement few of these ideas to improve further your software which is already now a great thing to use.
many greetings
Flip

**Reply by cdauth** [reply]
Great ideas! For 1., you can export your map to GeoJSON and then import the file in another map (by dragging it in or by using Tools → Open file). Then you will find your marker types under "Types" in the search box. Click on the plus to import them into your map. But I'm hoping to implement better ways to copy a map or parts of it.
The other ideas are also really good!

**Reply by cdauth** [reply]
For 2., marker types can now be reordered.

**Comment by Flip2376** [comment]
great. thank you. 

I was testing it yesterday by chance and realised that the language is also adapted...great step.

 
 

Gesendet: Freitag, 29. März 2024 um 12:32 Uhr
Von: "Candid Dauth" ***@***.***>
An: "FacilMap/facilmap" ***@***.***>
Cc: "Flip2376" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [FacilMap/facilmap] Moving position for tracking (Discussion #249)


 

For 2., marker types can now be reordered.

**Reply by Flip2376** [reply]
Hi,
after some months I would like to ask if the idea to implement moving marker (cars) is proceeding? 
many greetings
